packages/react-dom/src/client/ReactDOMFiberComponent.js

```diff
@@ -670,7 +668,7 @@ export function diffProperties(
   }
   for (propKey in nextProps) {
     const nextProp = nextProps[propKey];
-    const lastProp = lastProps != null ? lastProps[propKey] : undefined;
+    const lastProp = lastProps[propKey];
```

Comment:
Just thinking about moving carefully, is this change related to the text content change? Could we send this out separately?

I know that's tip toeing quite a bit. Just trying to figure out how to move forward with confidence :)

Reply:
Good eye :P yeah it's not related to the one change, just another thing I noticed. I did split it out by commit but we can do separate PR's

No harm in being intentional about the changes!